spring-cloud-sleuth-core/src/test/java/org/springframework/cloud/sleuth/instrument/web/SkipPatternProviderConfigTest.java

```diff
@@ -97,190 +116,163 @@ public void should_return_empty_when_no_endpoints() {
 
 	@Test
 	public void should_return_endpoints_without_context_path() {
-		ServerProperties properties = new ServerProperties();
-		WebEndpointProperties webEndpointProperties = new WebEndpointProperties();
-		EndpointsSupplier<ExposableWebEndpoint> endpointsSupplier = () -> {
-			ExposableWebEndpoint infoEndpoint = createEndpoint("info");
-			ExposableWebEndpoint healthEndpoint = createEndpoint("health");
-
-			return Arrays.asList(infoEndpoint, healthEndpoint);
-		};
-
-		Optional<Pattern> pattern = new TraceWebAutoConfiguration.ActuatorSkipPatternProviderConfig()
-				.skipPatternForActuatorEndpointsSamePort(properties,
-						webEndpointProperties, endpointsSupplier)
-				.skipPattern();
-
-		then(pattern).isNotEmpty();
-		then(pattern.get().pattern())
-				.isEqualTo("/actuator/(info|info/.*|health|health/.*)");
+		contextRunner.withConfiguration(UserConfigurations.of(EndpointConfig.class))
+				.withPropertyValues("management.endpoint.health.enabled=true",
+						"management.endpoint.info.enabled=true",
```

Comment:
Aren't the endpoints enabled by default? Meaning the result is the same without these properties.

Reply:
I believe they are, I could simplify it, thanks!